syntax = "proto3";

message FieldPB {
  string shoebox_name = 1;
  int64 object_id = 2;
  string field_name = 3;
}
